data/json/items/magazines.json

```diff
@@ -8,12 +8,62 @@
         "description" : "A standard capacity magazine for use with STANAG compatible rifles.",
         "material" : "steel",
         "volume" : 2,
-        "weight" : 110,
-        "to_hit" : -2,
+        "weight" : 360,
         "ammo_type" : "223",
         "capacity" : 30,
-        "reliability" : 10,
-        "reload_time" : 30
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags" : [ "COMPACT_MAG" ]
+    },
+    {   "type" : "MAGAZINE",
+        "id" : "stanag10",
+        "price" : 5000,
+        "name" : "Ruger 223 magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A compact 10-round magazine commonly used with semi-automatic STANAG compatible rifles.",
+        "material" : "steel",
+        "volume" : 1,
+        "weight" : 100,
+        "ammo_type" : "223",
+        "capacity" : 10,
+        "reliability" : 9,
+        "reload_time" : 100,
+        "flags" : [ "COMPACT_MAG" ]
+    },
+    {   "type" : "MAGAZINE",
+        "id" : "stanag50",
+        "price" : 9000,
+        "name" : "STANAG drum magazine",
```

Comment:
This what I had in mind:

![m249](https://cloud.githubusercontent.com/assets/12238606/12537195/0ff0aedc-c2b1-11e5-890d-2cbf49d4e680.jpg)

The 50-round capacity I want to keep the same so as to maintain overall balance. If you can think of a better name then that can be updated.

Above that I was intending to give the option for ammo belts. Whats a realistic maximum length, I've gone with 200 for now?